TYPO3.Neos/Documentation/References/Configuration/Configuration.rst

```diff
@@ -6,18 +6,22 @@
 Configuration Reference
 ========================
 
-Node tree ``loadingDepth``
-~~~~~~~~~~~~~~~~~~~~~~~~~~
+Navigation tree ``loadingDepth``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 ``loadingDepth`` defines the number of levels inside the node tree which shall be loaded eagerly, at start.
-If you have lots of nodes you should maybe reduce this number of elements inside ``Settings.yaml``::
+A similar setting is available for the structure tree.
+
+If you have lots of nodes can reduce the number of levels inside ``Settings.yaml`` to speed up page loading::
```

Comment:
If you have lots of nodes `you` can reduce the number of levels inside ``Settings.yaml`` to speed up page loading::